recipes/osprofiler/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "osprofiler" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: eb18651bc58ae27c11cbdd201ff373254ea92750e29c62df065b2bc4c6910d48
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - osprofiler = osprofiler.cmd.shell:main
+
+requirements:
+  host:
+    - pbr >=2.0
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - netaddr >=0.7.18
+    - oslo.concurrency >=3.26.0
+    - oslo.serialization >=2.18.0
+    - oslo.utils >=3.33.0
+    - prettytable >=0.7.2
```

Comment:
```suggestion
    - prettytable >=0.7.2,<0.8
```

Reply:
I am not in favour of this. The openstack people have a bad habit of adding upper version constraints in requirements.txt to exclude future major updates and then never seem to change them again, even though they work with these newer versions of the dependencies.